api/net/Dockerfile.openshift

```diff
@@ -1,12 +1,13 @@
 ARG BUILD_CONFIGURATION=Release
+# TODO: We will need to update the following to use .net 7 when the information is available
 FROM image-registry.apps.silver.devops.gov.bc.ca/9b301c-tools/aspnet:6.0 AS base
```

Comment:
This needs to be updated to `7.0`.  I have pushed the image to Openshift already.

Reply:
Updated, thanks